docs/plugins.md

```diff
@@ -0,0 +1,351 @@
+      'When integrated into your website effectively, cats deliver delight and bemusement.',
+    auditRefs: [{id: 'has-cat-images-id', weight: 1}],
+  },
+};
+```
+
+#### Custom Audits
+
+These files contain the logic that will generate results for the Lighthouse report. An audit is a class with two important properties:
+
+1. `meta` - This contains important information about how the audit will be referenced and how it will be displayed in the HTML report.
+2. `audit` - This is a function that should return the audit's results. See [API > Plugin Audits](#plugin-audits).
+
+**Example `audits/has-cat-images.js`**
+
+```js
+const Audit = require('lighthouse').Audit;
+
+class CatAudit extends Audit {
+  static get meta() {
+    return {
+      id: 'has-cat-images-id',
+      title: 'Page has least one cat image',
+      failureTitle: 'Page does not have at least one cat image',
+      description:
+        'Pages should have lots of cat images to keep users happy. ' +
+        'Consider adding a picture of a cat to your page improve engagement.',
+      requiredArtifacts: ['ImageElements'],
+    };
+  }
+
+  static audit(artifacts) {
+    // Artifacts requested in `requiredArtifacts` above are passed to your audit.
+    // See the "API -> Plugin Audits" section below for what artifacts are available.
+    const images = artifacts.ImageElements;
+    const catImages = images.filter(image => image.src.toLowerCase().includes('cat'));
+
+    return {
+      // Give users a 100 if they had a cat image, 0 if they didn't.
+      score: catImages.length > 0 ? 1 : 0,
+      // Also return the total number of cat images that can be used by report JSON consumers.
+      numericValue: catImages.length,
+    };
+  }
+}
+
+module.exports = CatAudit;
+```
+
+## API
+
+### Plugin Config
+
+The plugin config file (see `plugin.js` in the [example](#pluginjs) and [recipe](./recipes/lighthouse-plugin-example/plugin.js)) is a subset of the available [configuration](./configuration.md) for full custom Lighthouse config files.
+
+A plugin config is an object that has at least two properties: `audits` and `category`.
+
+#### `audits`
+
+Defines the new audits the plugin adds. It is an array of string paths to the audit files. Each path should be treated as an absolute string a user of your module might pass to `require`, so use paths of the form `lighthouse-plugin-<your plugin>/path/to/audits/audit-file.js`.
+
+**Type**: `Array<{path: string}>`
+
+#### `category`
+
+Defines the display strings of the plugin's category and configures audit scoring and grouping. It is an object with at least two properties `title` and `auditRefs`.
+
+- `title: string` **REQUIRED** - The display name of the plugin's category in the report.
+- `description: string` _OPTIONAL_ - A more detailed description of the category's purpose.
+- `manualDescription: string` _OPTIONAL_ - A more detailed description of all of the manual audits in a plugin. Only use this if you've added manual audits.
```

Comment:
ha, I half feel like we shouldn't even mention this... ("what's a manual audit?")

Reply:
Heh, yeah I don't think we should encourage them for sure :) but feels like we should at least document what it is